src/ProjectTemplates/test/WebApiTemplateTest.cs

```diff
@@ -28,7 +28,6 @@ public WebApiTemplateTest(ProjectFactoryFixture factoryFixture, ITestOutputHelpe
 
         [ConditionalFact]
         [SkipOnHelix("Cert failures", Queues = "OSX.1014.Amd64;OSX.1014.Amd64.Open")]
-        [QuarantinedTest]
```

Comment:
No issue or logs associated with the failure that caused this to be quarantined are available.
It was quarantined 1.5 months ago and has had no failures in at least 30 days.

Propose un-quarantining and monitoring it for future failures.